src/elements/Badge/Badge.stories.tsx

```diff
@@ -58,6 +62,18 @@ WithStatus.args = {
     status: BadgeStatus.Positive,
 };
 
+export const WithHexStatusColor = BadgeTemplate.bind({});
+
+WithStatus.argTypes = {
+    status: {
+        control: { type: "text" },
```

Comment:
somehow the control in storybook is still a select for the BadgeStatus and not a text input

Reply:
agreed: 8e01b16